scripts/lbnl_time.nhc

```diff
@@ -0,0 +1,31 @@
+# LBNL NHC - Time checks
+#
+# Andy Georges <andy.georges@ugent.be>
+# 15 December 2010
+#
+
+
+# Checks the drift from the ptpd status file and compares it to the given threshold
+function check_ptpd_drift() {
+    local DRIFT=0
+
+    if [ ! -f "$NHC_PTPD_STATUS" ]; then
+        die 1 "$FUNCNAME: No $NHC_PTPD_STATUS file found"
+        return 1
+    fi
+    DRIFT=$(grep "Offset from Master" < "$NHC_PTPD_STATUS" | tr -s " " | cut -d" " -f5)
+    dbg "$FUNCNAME: drift is $DRIFT"
+
+    if [ -z "${DRIFT+x}" ]; then
+            die 1 "$FUNCNAME: No drift line found in $NHC_PTPD_STATUS"
+            return 1
+    fi
+
+    if (( $(echo "${DRIFT#-} > $NHC_PTPD_MAX_DRIFT " | bc -l) )); then
```

Comment:
why not `-gt`?

Reply:
`-gt` is for integer comparison afaik.

The status file has

~~~~
Offset from Master :  0.000015214 s, mean  0.000007196 s, dev  0.000037799 s
~~~~